src/apb/engine.py

```diff
@@ -726,8 +726,6 @@ def retrieve_test_result(name, namespace):
 def broker_request(broker, service_route, method, **kwargs):
     if broker is None:
         broker = get_asb_route()
-    else:
-        broker = "%s/ansible-service-broker" % broker
```

Comment:
From apb push we were already setting broker so this was appending a second suffix.